verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -253,3 +254,24 @@ void ensure_nondet_allocate_cmm_vtable_members(struct aws_cryptosdk_cmm_vt *vtab
         vtable->name = ensure_c_str_is_allocated(max_len);
     }
 }
+
+struct aws_cryptosdk_session *ensure_session_attempt_allocation(size_t max_len) {
+    struct aws_cryptosdk_session *session = malloc(sizeof(struct aws_cryptosdk_session));
+    if (session) {
+        session->alloc       = nondet_bool() ? NULL : can_fail_allocator();
+        session->cmm         = malloc(sizeof(struct aws_cryptosdk_cmm));
+        session->header_copy = malloc(sizeof(uint8_t));
+        session->alg_props   = ensure_alg_properties_attempt_allocation(max_len);
+        session->signctx     = ensure_nondet_sig_ctx_has_allocated_members();
```

Comment:
```c
    struct aws_allocator *alloc;
    struct aws_cryptosdk_cmm *cmm;
    uint8_t *header_copy;
    const struct aws_cryptosdk_alg_properties *alg_props;
    struct aws_cryptosdk_sig_ctx *signctx;
```
Makes sure all are properly allocated and, if they also have pointers, their pointers are properly allocated.